Scripts/Interfaces/Entity/IManipulator.cs

```diff
@@ -13,7 +21,7 @@ public interface IManipulator
 		/// Delegate that processes the translation, using the vector3 passed in
 		/// Caller also provides the ray origin that is doing the action, and whether or not this translation is axis-constrained
```

Comment:
Probably want to update the comments to reflect the change from bool to ConstrainedAxis. Something to the effect of "and constrained axes"